formSkeleton/forms/Overzicht-vergoedingen-en-presentiegelden/form.ttl

```diff
@@ -0,0 +1,49 @@
+########### Overzicht vergoedingen en presentiegelden ###########
+
+# Conditionally add the form fields to the "type dossier" field
+fields:0827fafe-ad19-49e1-8b2e-105d2c08a54a form:hasConditionalFieldGroup fields:26b8392b-4097-4e99-8fec-832d063d316b.
+
+# Only add the fields if the user selected the "Overzicht vergoedingen en presentiegelden" option
+fields:26b8392b-4097-4e99-8fec-832d063d316b a form:ConditionalFieldGroup ;
+    mu:uuid "26b8392b-4097-4e99-8fec-832d063d316b";
+    form:conditions
+      [ a form:SingleCodelistValue ;
+        form:grouping form:Bag ;
+        sh:path rdf:type ;
+        form:conceptScheme <http://lblod.data.gift/concept-schemes/71e6455e-1204-46a6-abf4-87319f58eaa5> ;
+        form:customValue <https://data.vlaanderen.be/id/concept/BesluitDocumentType/365d561c-57c7-4523-af04-6e3c91426c56>
+      ] ;
+    form:hasFieldGroup fieldGroups:8a07bf7d-c99f-47fa-9bcf-a4b7d221ac28 .
+
+fieldGroups:8a07bf7d-c99f-47fa-9bcf-a4b7d221ac28 a form:FieldGroup ;
+    mu:uuid "8a07bf7d-c99f-47fa-9bcf-a4b7d221ac28" ;
+
+    form:hasField
+      ### Bestanden
+      fields:1d8f3fa0-a908-494c-8cd3-48035f581936,
```

Comment:
This will show the "Indien geen links" label due to a bug in the[ form field component logic](https://github.com/lblod/ember-submission-form-fields/blob/65cac39e61fe1a48fe8ce7020875ccbcd3143b62/addon/components/rdf-input-fields/files/edit.js#L45-L51).

That getters always returns true because the remoteUrl displaytype is always part of the form (even when not visible in a subform).

Awaiting feedback to know the next steps.

Reply:
We can add the link field as well :+1: